src/test/java/api/loans/DeclareLostAPITests.java

```diff
@@ -102,18 +102,17 @@ public void canDeclareItemLostWithoutComment() {
 
   @Test
   public void cannotDeclareItemLostForAClosedLoan() {
-    UUID loanId = UUID.fromString(loanJson.getString("id"));
-    DateTime dateTime = DateTime.now();
+    final InventoryItemResource item = itemsFixture.basedUponSmallAngryPlanet();
 
+    final IndividualResource checkIn = loansFixture.checkOutByBarcode(item, usersFixture.jessica());
```

Comment:
The operation being performed is a check out, yet the variable is called check in. Could you help me understand this discrepancy?